conf.py

```diff
@@ -33,5 +33,7 @@
     }
 }
 
+jira_uri_template = "https://ls.st/{ticket}"
```

Comment:
How does this work?

Reply:
Oh I forgot about this configuration! Thanks for finding it. I'll make a documenteer release to update the default.

Reply:
I checked that the RFC-X and DM-X forwards from ls.st resolve.